Client/Cliqz/Frontend/Browser/Dashboard/FavoritesViewController.swift

```diff
@@ -21,5 +21,10 @@ class FavoritesViewController: HistoryViewController {
     override func createDataSource(_ profile: Profile) -> HistoryDataSource {
         return FavoritesDataSource(profile: profile)
     }
+    
+    override func getEmptyHistroyText() -> String {
```

Comment:
getEmptyHistroyText would be better to have more generic name, as historytext in FavoritesViewController is confusing. Maybe somothnig like emptyViewText...